hoomd/md/PotentialPairGPU.h

```diff
@@ -79,6 +79,10 @@ class PotentialPairGPU : public PotentialPair<evaluator>
         }
 
     protected:
+    virtual void computeTailCorrection()
+        {
+        PotentialPair<evaluator>::computeTailCorrection();
+        }
```

Comment:
```suggestion
```
Base class methods will be called automatically. We only need to implement this method specifically here if we add a GPU optimized implementation.

Reply:
That's what I had originally thought, but not declaring that here led to compilation errors of the sort reported [here](https://stackoverflow.com/questions/9941987/there-are-no-arguments-that-depend-on-a-template-parameter). This error is resolved by using the `this` pointer; see [this commit](https://github.com/glotzerlab/hoomd-blue/pull/1138/commits/44914300670fcc690a4df6c8e9e4280c8cb161fa) for the updated code.